docs/troubleshooting_pi-server_integration.rst

```diff
@@ -0,0 +1,180 @@
+
+|img_007|
+
+*selecting the instance*
+
+|img_008|
+
+*selecting the database*
+
+|img_009|
+
+proceed with the drill down operation up to the desired level/asset.
+
+Error messages an the related causes
+------------------------------------
+
+Some error messages and the related causes:
+
+.. list-table::
+    :widths: 50 50
+    :header-rows: 1
+
+    * - Message
+      - Cause
+    * - North_Readings_to_PI[20884]: WARNING: Error in retrieving the PIWebAPI version, The **PI Web API server is not reachable**, verify the network reachability
+      - Fledge is not able to reach the machine in which PI-Server is running due to a network problem of a firewall restriction.
+    * - North_Readings_to_PI[5838]: WARNING: Error in retrieving the PIWebAPI version, **503 Service Unavailable**
+      - Fledge is capable to reach the machine in which PI-Server is executed but the PI Web API is not running.
+    * - North_Readings_to_PI[24485]: ERROR: Sending JSON data error : **Container not found**. 4273005507977094880_1measurement_sin_4816_asset_1 - WIN-4M7ODKB0RH2:443 /piwebapi/omf
+      - Fledge is able to interact with PI Web API but there is an attempt to store data in a PI Point that does not exist.
+
+Possible solutions to common situations
+---------------------------------------
+
+- **Recreate a single or a sets of PI-Server objets and send again all the data for them to the PI Server on the Asset Framework hierarchy level**
+    procedure:
+        - disable the 1st north instance
+        - delete properly the objects on the PI Server, AF + Data archive, that we intend to recreate or were eventually partially deleted
+        - create a new **DISABLED** north instance using a new/unique name and having the same AF hierarchy of the 1st north instance
+        - install *fledge-filter-asset* on the new north instance, available at https://github.com/fledge-iot/fledge-filter-asset
+        - configure *fledge-filter-asset* with a rule like the following one
+        - enable the 2nd north instance
+        - let the  2nd north instance send the desired amount of data and then disable it
+        - enable the 1st north instance
+
+    consideration:
+        - the 2nd north instance will be used only to recreate the objects and resend the data
+        - the 2nd north instance will send again all the data available for the specified *included* assets
+        - there will some data duplicated for the recreated assets because parte of the information will be managed by both the north instances.
+
+.. code:: json
+
+    {
+      "rules": [
+        {
+          "asset_name": "asset_4",
+          "action": "include"
+        }
+      ],
+      "defaultAction": "exclude"
+    }
+
+- **Recreate all the PI-Server objets and send again all the data to the PI Server on a different Asset Framework hierarchy level**
```

Comment:
Recreate all the PI-Server objets and resend all the data to the PI Server to a different Asset Framework hierarchy level